src/codegen/object.rs

```diff
@@ -47,6 +47,24 @@ pub fn generate(
             )?;
         }
 
+        if !analysis.builder_properties.is_empty() {
+            // generate builder method that returns the corresponding builder
+            let builder_name = format!("{}Builder", analysis.name);
+            writeln!(
+                w,
+                "
+            // rustdoc-stripper-ignore-next
+            /// Creates a new builder-style object to manufacture a [`{name}`]
```

Comment:
```suggestion
            /// Creates a new builder-style object to construct a [`{name}`]
```

Maybe? We call `new` functions "constructors", not manufacturers ;)

Reply:
changed :-)